src/components/ATATFileUpload.vue

```diff
@@ -288,6 +288,14 @@ export default class ATATFileUpload extends Vue {
    */
   private removeInvalidFiles(files: FileList): void {
     let _validFiles = Array.from(files || []).filter((vFile) => {
+      const restrictedNames = [
```

Comment:
pass `restrictedNames` array from `UploadJAMRRDocuments.vue` on the `ATATFileUpload` component